lib/joke.rb

```diff
@@ -0,0 +1,23 @@
+class Joke
+
+  attr_reader :joke, :number, :answer
+
+  def initialize(number, joke, answer)
+    @number = number
+    @joke = joke
+    @answer = answer
+  end
+
+  def id
```

Comment:
I see where you were coming from with this `id`, `setup`, and `punchline` methods, but you don't want to hardcode these values-- remember, this is a template for creating all jokes, and not every joke is going to have an id of 1, a setup of "Why did the strawberry cross the road?", and a punchline of "Because his mother was in a jam."  By passing these in as arguments and setting them in the initialize method, you take care of both setting those values and making sure they're flexible. So you really just need your `def initialize` method here.